telegram/constants.py

```diff
@@ -223,7 +224,7 @@
 
 POLL_REGULAR: str = 'regular'
 POLL_QUIZ: str = 'quiz'
-
+POLL_QUESTION_LENGTH: int = 300
```

Comment:
maybe rather `MAX_POLL_QUESTION_LENGTH` in accordance with the other `MAX_*` constants? Also, while we're at it, maybe also add `MAX_POLL_OPTION_LENGTH`?

Reply:
Oh. Yeah.